### PR TITLE
[All] Cleanup update flag handling

### PR DESCRIPTION
**Pull request by NicolasDievart**

### CODE REVIEW

File: manala.npm/README.md [hunk 1]
```diff
@@ -32,13 +32,19 @@ Using ansible galaxy requirements file:
...
 ## Role Variables
 
-| Name                  | Default | Type  | Description       |
-| --------------------- | ------- | ----  | ----------------- |
-| `manala_npm_packages` | [ ]     | Array | Npm packages list |
+| Name                  | Default | Type    | Description                                                         |
+| --------------------- | ------- | ------- | ------------------------------------------------------------------- |
+| `manala_npm_packages` | [ ]     | Array   | Npm packages list                                                   |
+| `manala_npm.update`   | false   | Boolean | Update to the latest version of npm                                 |
+| `manala_npm.state`    | -       | String  | The version of npm, could be `latest` to always have the latest one |
```
Comment (nervo) [inline manala.npm/README.md]: Wait, where the hell this variable comes from ?

File: manala.npm/README.md [hunk 2]
```diff
@@ -32,13 +32,19 @@ Using ansible galaxy requirements file:
...
+| `manala_npm.global`   | true    | Boolean | Install npm globally                                                |
```
Comment (nervo) [inline manala.npm/README.md]: Same here :)

File: manala.npm/README.md (hunk 1 above)
Comment (NicolasDievart) [reply manala.npm/README.md]: My bad, thought you could overwrite it from there https://github.com/manala/ansible-roles/blob/master/manala.npm/tasks/packages.yml#L6

Comment (NicolasDievart) [reply manala.npm/README.md]: I'll remove them then

**Comment by NicolasDievart** [comment]
@nervo Your comments are fixed with my last commit, sorry for the misinterpretation

**Comment by NicolasDievart** [comment]
@nervo Any news from this ? It's just doc

**Comment by nervo** [comment]
Ok, @NicolasDievart i took the opportunity to settle and clarify update flag across roles

**Comment by NicolasDievart** [comment]
@nervo No entry in the changelog ?

**Comment by nervo** [comment]
@NicolasDievart Nope, this is just a matter of documentation.

**Comment by nervo** [comment]
(and a bit of laziness)

**Pull request merged**